test/JDBC/expected/BABEL-328.out

```diff
@@ -165,15 +165,63 @@ GO
 ~~ERROR (Message: syntax error near 'Employee' at line 4 and character position 6)~~
 
 
+-- chaining apply calls
 SELECT * FROM Department D
 OUTER Employee E
 GO
 ~~ERROR (Code: 33557097)~~
 
-~~ERROR (Message: syntax error near 'Employee' at line 2 and character position 6)~~
+~~ERROR (Message: syntax error near 'Employee' at line 3 and character position 6)~~
 
 
 DROP TABLE [Employee]
 GO
 DROP TABLE [Department]
 GO
+
+create table t1 (a int, b int)
+create table t2 (c int, d int)
+insert into t1 values (1, 1),(2, 2)
+insert into t2 values (3, 3),(4, 4)
+go
+~~ROW COUNT: 2~~
+
+~~ROW COUNT: 2~~
+
+
+select * from t1 outer apply t2 outer apply (values (5,5),(6,6)) t3 (e, f)
+go
+~~START~~
+int#!#int#!#int#!#int#!#int#!#int
+1#!#1#!#3#!#3#!#5#!#5
+1#!#1#!#3#!#3#!#6#!#6
+1#!#1#!#4#!#4#!#5#!#5
+1#!#1#!#4#!#4#!#6#!#6
+2#!#2#!#3#!#3#!#5#!#5
+2#!#2#!#3#!#3#!#6#!#6
+2#!#2#!#4#!#4#!#5#!#5
+2#!#2#!#4#!#4#!#6#!#6
+~~END~~
+
+
+select * from t1 cross apply t2 cross apply (values (5,5),(6,6)) t3 (e, f)
+go
+~~START~~
+int#!#int#!#int#!#int#!#int#!#int
+1#!#1#!#3#!#3#!#5#!#5
+1#!#1#!#4#!#4#!#5#!#5
+1#!#1#!#3#!#3#!#6#!#6
+1#!#1#!#4#!#4#!#6#!#6
+2#!#2#!#3#!#3#!#5#!#5
+2#!#2#!#4#!#4#!#5#!#5
+2#!#2#!#3#!#3#!#6#!#6
+2#!#2#!#4#!#4#!#6#!#6
+~~END~~
+
+
+drop table t1
+go
+
+drop table t2
```

Comment:
1. Please follow our object name conventions, e.g., `babel_328_vu_xxx`.
2. Our test strategy is to add upgrade tests unless it is unnecessary or impossible. So, please consider to move this to upgrade tests using views.

Reply:
Moved all tests to upgrade tests